client/src/utils/slate/handleStationaryCursor.ts

```diff
@@ -0,0 +1,25 @@
+import { Change } from "../../components/BlockEditor/types";
+
+export function handleStationaryCursor(change: Change) {
```

Comment:
I think the name of this could be more clear. "handleStationaryCursor" really doesn't tell me much about what's going on; from the above use it seems to be doing some DOM manipulation without really returning anything important. Does this need to return "change", if in the one case it's being used, it doesn't accept that return? (Just to be more obvious what's going on.)

Maybe "adjustCursorIfAtEdge"